astropy/time/tests/test_basic.py

```diff
@@ -2317,3 +2317,13 @@ def test_location_init_fail():
     with pytest.raises(ValueError,
                        match='cannot concatenate times unless all locations'):
         Time([tm, tm2])
+
+
+def test_linspace():
+    t1 = Time('2020-01-01 00:00:00')
+    t2 = Time('2020-01-01 01:00:00')
+
+    ts = np.linspace(t1, t2, 3)
+    assert ts[0] == t1
+    assert u.allclose((ts[1] - Time('2020-01-01 00:30:00')).to_value(u.s), 0*u.s)
```

Comment:
Excepting the error this is currently raising, is this the best way to compare two times to within a delta?

Reply:
I've tended to do `assert abs(t1 - t2) < xxx * u.ns`, but your construct should work except that it should be `.to(u.s)` rather than `.to_value(u.s)` to ensure both sides are `Quantity`.

...

Though actually, I now remember we added an `.isclose` method, so `assert t1.isclose(t2, atol=xxx*u.ns)` would work too!

And, of course, once you have implemented `np.isclose` and `np.allclose` for `__array_function__`, you can use those directly...